solutions/devsprint-ana-nogal-1/DeliveryApp/Screens/LoadingView/LoadingView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LoadingView.swift
+//  DeliveryApp
+//
+//  Created by Luiza on 26/04/22.
+//
+
+import UIKit
+
+final class LoadingView: UIView {
+
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Buscando endereços..."
```

Comment:
O texto esta hardcoded... o melhor será inicializar a vazio.
```suggestion
        label.text = ""
```

Reply:
Done✅